upgrade/network/chn_enable.md

```diff
@@ -0,0 +1,856 @@
+   playbook = "cos-compute.yml"
+
+   [[layers]]
+   cloneUrl = "https://api-gw-service-nmn.local/vcs/cray/sma-config-management.git"
+   commit = "2219ca094c0a2721f3bf52f5bd542d8c4794bfed"
+   name = "sma-base-config"
+   playbook = "sma-base-config.yml"
+
+   [[layers]]
+   cloneUrl = "https://api-gw-service-nmn.local/vcs/cray/sma-config-management.git"
+   commit = "2219ca094c0a2721f3bf52f5bd542d8c4794bfed"
+   name = "sma-ldms-ncn"
+   playbook = "sma-ldms-ncn.yml"
+
+   [[layers]]
+   cloneUrl = "https://api-gw-service-nmn.local/vcs/cray/slurm-config-management.git"
+   commit = "0982661002a857d743ee5b772520e47c97f63acc"
+   name = "slurm master"
+   playbook = "site.yml"
+
+   [[layers]]
+   cloneUrl = "https://api-gw-service-nmn.local/vcs/cray/pbs-config-management.git"
+   commit = "874050c9820cc0752c6424ef35295289487acccc"
+   name = "pbs master"
+   playbook = "site.yml"
+
+   [[layers]]
+   cloneUrl = "https://api-gw-service-nmn.local/vcs/cray/analytics-config-management.git"
+   commit = "d4b26b74d08e668e61a1e5ee199e1a235e9efa3b"
+   name = "analytics integration"
+   playbook = "site.yml"
+
+   [[layers]]
+   cloneUrl = "https://api-gw-service-nmn.local/vcs/cray/cos-config-management.git"
+   commit = "dd2bcbb97e3adbfd604f9aa297fb34baa0dd90f7"
+   name = "cos-compute-last-integration-2.3.75"
+   playbook = "cos-compute-last.yml"
+
+   [[layers]]
+   cloneUrl = "https://api-gw-service-nmn.local/vcs/cray/uan-config-management.git"
+   commit = "359611be2f6893ddd0020841b73a3d4924120bb1"
+   name = "chn-uan-cn"
+   playbook = "site.yml"
+   ```
+
+5. (`ncn-m001#`) Check that CFS configuration of the compute node completes successfully.
+
+   Updating the CFS configuration will cause CFS to schedule the nodes for configuration. Run the following command to verify this has occurred.
+
+   ```bash
+   cray cfs components describe --format toml x1000c5s1b0n1
+   ```
+
+   Example output:
+
+   ```toml
+   configurationStatus = "pending"
+   desiredConfig = "cos-config-full-2.3-integration"
+   enabled = true
+   errorCount = 0
+   id = "x1000c5s1b0n1"
+   state = []
+
+   [tags]
+   ```
+
+   `configurationStatus` should change from `pending` to `configured` once CFS configuration of the node is complete.
+
+For more information on managing node with CFS, see the [Configuration Management](../../README.md#configuration-management) documentation.
+
+## Cleanup Phase
+
+**Recommend** copying and storing all data in the ${CLEANUPDIR} off-system in a version control repository is highly recommended.
+
+### Remove CAN from SLS
+
+1. (`ncn-m001#`) Move to the update directory.
+
+   ```bash
+   cd ${CLEANUPDIR}
+   ```
+
+2. (`ncn-m001#`) Set the directory location for the `SLS` `CHN` script
+
+   ```bash
+   export SLS_CHN_DIR=/usr/share/doc/csm/upgrade/scripts/sls
+   ```
+
+3. (`ncn-m001#`) Remove `CAN` to `SLS` data.
+
+   ```bash
+   ${SLS_CHN_DIR}/sls_del_can.py \
+      --sls-input-file ${UPDATEDIR}/sls_file_with_chn.json \
```

Comment:
What if CHN was added for computes above?   It think this will end up removing those CHN IPs from SLS, right?

Reply:
yup. good find.